app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        Person.people[name] = self
+
+    def __repr__(self) -> str:
+        return f"Person(name={self.name}, age={self.age})"
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+
+    for data in people:
+        person = Person(data["name"], data["age"])
+        persons.append(person)
+
+    for data in people:
+        person = Person.people[data["name"]]
+
+        if "wife" in data and data["wife"] is not None:
+            wife_name = data["wife"]
+            if wife_name in Person.people:
+                person.wife = Person.people[wife_name]
```

Comment:
The code assumes that the `wife` name exists in the `Person.people` dictionary. If the name is not found, it will silently fail to set the `wife` attribute. Consider adding error handling or logging to handle cases where the `wife` name is not found.